### PR TITLE
Clarifies Bone Repair Options

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR clarifies what the two different bone repair options are for.
~this PR removes the duplicate bone repair option, "Mend closed bone", from the bone gel.~

## changes
- renames "Begin bone repair" to "Repair Broken Bone" to better reflect its purpose.
- renames "Mend closed bone" to "Repair Sawed Bone" to better reflect its purpose.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!bugfix

**Comment by SleepyGemmy** [comment]
!review

**Comment by Melariara** [comment]
Just came here to say, its not a duplicate. One of them is for fixing bones from breaking, and the other is for resealing the bones from carving them open, IE opening the chest and closing it again.

**Comment by FayeLovesU** [comment]
i came here 2 say what mel was gonna say. maybe the functions could be merged? it's rare i'm fixing chest / skull fracs without also wanting to close the bone.

**Comment by SleepyGemmy** [comment]
"begin bone surgery" does the same thing afaik.

**Comment by SleepyGemmy** [comment]
"begin bone repair" does the same thing afaik.

**Comment by Melariara** [comment]
On live? No.

image

Begin bone repair starts the surgery to repair the broken bone. Applying the gel again does the fixing of the bending.

**Review by KingOfThePing — APPROVED** [review]

**Review by alsoandanswer — DISMISSED** [review]
Keep it as FixOVein.

The wiki's going to have to rename everything again if so.

Something like this should *not* be in a bugfix.

**Comment by SleepyGemmy** [comment]
true, but im keeping the path name.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
We don't need more incoherent path names. Change the path to fixovein again.

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/surgery/encased.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 //				GENERIC	RIBCAGE SURGERY							//
 //////////////////////////////////////////////////////////////////
 /decl/surgery_step/open_encased
-	name = "Saw through bone"
+	name = "Saw Through Bone"
```
Comment (NonQueueingMatt) [inline code/modules/surgery/encased.dm:6]: This capitalisation is inconsistent with all other surgeries. Keep it "Saw through bone" and do the same for the rest you modified.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/surgery/encased.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/modules/surgery/encased.dm:6]: surgical steps were mixed between non-titlecase and titlecase. all surgical steps now use titlecase, as per 96e0373.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**